ui/raidboss/data/05-shb/raid/e5s.js

```diff
@@ -7,6 +7,206 @@
+      regexKo: Regexes.startsUsing({ id: '4BAA', source: '라무', capture: false }),
+      infoText: {
+        en: 'Fury\'s Bolt',
+        ko: '라무 강화',
+      },
+    },
+    {
+      id: 'E5S Judgment Volts',
+      regex: Regexes.startsUsing({ id: '4BB5', source: 'Ramuh', capture: false }),
+      regexDe: Regexes.startsUsing({ id: '4BB5', source: 'Ramuh', capture: false }),
+      regexFr: Regexes.startsUsing({ id: '4BB5', source: 'Ramuh', capture: false }),
+      regexJa: Regexes.startsUsing({ id: '4BB5', source: 'ラムウ', capture: false }),
+      regexKo: Regexes.startsUsing({ id: '4BB5', source: '라무', capture: false }),
+      infoText: {
+        en: 'aoe',
+        de: 'AoE',
+        fr: 'Dégâts de zone',
+        cn: 'AOE',
+        ko: '전체 공격',
+      },
+    },
+    {
+      id: 'E5S Stepped Leader',
+      regex: Regexes.startsUsing({ id: '4BC6', source: 'Ramuh', capture: false }),
+      regexDe: Regexes.startsUsing({ id: '4BC6', source: 'Ramuh', capture: false }),
+      regexFr: Regexes.startsUsing({ id: '4BC6', source: 'Ramuh', capture: false }),
+      regexJa: Regexes.startsUsing({ id: '4BC6', source: 'ラムウ', capture: false }),
+      regexKo: Regexes.startsUsing({ id: '4BC6', source: '라무', capture: false }),
+      preRun: function(data) {
+        data.leaderCount = data.leaderCount || 0;
+      },
+      alertText: function(data) {
+        if (data.leaderCount %2 == 0) {
+          return {
+            en: 'Ready Spread',
+            ko: '산개 준비',
+          };
+        } else if (data.leaderCount %2 == 1) {
+          return {
+            en: 'donut AoE',
+            ko: '도넛 장판',
+          };
+        }
+      },
+      run: function(data) {
+        data.leaderCount++;
+      }
+    },
+    {
+      id: 'E5S Stepped Leader Spread',
+      regex: Regexes.ability({ id: '4BC6', source: 'Ramuh', capture: false }),
+      regexDe: Regexes.ability({ id: '4BC6', source: 'Ramuh', capture: false }),
+      regexFr: Regexes.ability({ id: '4BC6', source: 'Ramuh', capture: false }),
+      regexJa: Regexes.ability({ id: '4BC6', source: 'ラムウ', capture: false }),
+      regexKo: Regexes.ability({ id: '4BC6', source: '라무', capture: false }),
+      alarmText: function(data) {
+        if (data.leaderCount %2 == 0) {
+          return {
+            en: 'Spread',
+            de: 'Verteilen',
+            fr: 'Ecartez-vous',
+            ja: '散開',
+            cn: '分散',
+            ko: '산개',
+          };
+        }
+      },
+    },
+    {
+      id: 'E5S Crippling Blow',
+      regex: Regexes.ability({ id: '4BCA', source: 'Ramuh', capture: false }),
+      regexDe: Regexes.ability({ id: '4BCA', source: 'Ramuh', capture: false }),
+      regexFr: Regexes.ability({ id: '4BCA', source: 'Ramuh', capture: false }),
+      regexJa: Regexes.ability({ id: '4BCA', source: 'ラムウ', capture: false }),
+      regexKo: Regexes.ability({ id: '4BCA', source: '라무', capture: false }),
+      alertText: function(data, matches) {
```

Comment:
`response: Responses.tankBuster()`

Reply:
Hmmmm... how do I change this to that?
```javascript
alertText: function(data, matches) {
        if (matches.target == data.me) {
          return {
            en: 'Tank Buster on YOU',
            de: 'Tankbuster auf DIR',
            fr: 'Tank Buster sur VOUS',
            ko: '탱버 대상자',
          };
        }
      },
```
to
```javascript
alertText: function(data, matches) {
        if (matches.target == data.me) {
          return Responses.tankBuster();
        }
      },
```
does this work?

Reply:
I was just suggesting `response: Responses.tankBuster()` on its own, with no alertText.  Maybe add a condition that it is on you or that you're a healer, like https://github.com/quisquous/cactbot/pull/1064/files#diff-7cf55164a059021c14004927f7f84f4eR20